velox/exec/RowContainer.h

```diff
@@ -192,6 +192,17 @@ class RowContainer {
       RowColumn col,
       VectorPtr result);
 
+  // Copies the values from the array pointed to by 'rows' at 'col' into
+  // 'result' (starting at 'resultOffset') for the rows at positions in
+  // the 'offsets' array. If an entry in 'rows' is null, sets corresponding
+  // row in 'result' to null.
+  static void extractColumnAtOffsets(
+      const char* const* rows,
+      const BufferPtr& offsets,
```

Comment:
I wonder if it would be clearer to rename this variable to rowNumbers and drop "AtOffsets" suffix from the method name.

Are there any restrictions on the content of 'offsets'? Is it allowed to specify duplicate row numbers? Can they be specified our of order, e.g. 10, 5, 25, 7, etc.? If all these things are allowed, then let's add test cases to ensure these work correctly.

Reply:
Yes, all those cases are allowed. Will add new testcases.